primitives/bridge/src/lib.rs

```diff
@@ -86,8 +86,8 @@ impl Command {
     pub fn index(&self) -> u8 {
         match self {
             // Starting from 32 to keep compatibility with Snowbridge Command enum
-            Command::Test { .. } => 32,
-            Command::ReportRewards { .. } => 33,
+            Command::Test { .. } => 12,
+            Command::ReportRewards { .. } => 13,
```

Comment:
We talked about just adding dummy variants to the solidity enum, like

Variant29, Variant30, Variant31, etc

Reply:
Okay. I have reverted this change.

Reply:
*fixed*